core/src/test/java/org/apache/calcite/test/TypeCoercionTest.java

```diff
@@ -545,16 +545,16 @@ assert equals(result1, expected)
     expr("'12.3'/cast(5 as double)")
         .columnType("DOUBLE NOT NULL");
     expr("'12.3'/5.1")
-        .columnType("DECIMAL(19, 18) NOT NULL");
+        .columnType("DECIMAL(19, 8) NOT NULL");
     expr("12.3/'5.1'")
-        .columnType("DECIMAL(19, 0) NOT NULL");
+        .columnType("DECIMAL(19, 8) NOT NULL");
```

Comment:
here comes the 8, by the way, the origin scale 0 is not reasonable
![image](https://user-images.githubusercontent.com/15643702/95651278-d590ef00-0b1b-11eb-893c-9b320f5588cc.png)